tests/packages/link-library-in-subproject/subprojects/bar/meson.build

```diff
@@ -0,0 +1,36 @@
+# SPDX-FileCopyrightText: 2022 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project('bar', 'c', version: '1.2.3', meson_version: '>= 1.3.0')
+
+if get_option('default_library') == 'shared' and meson.get_compiler('c').get_id() in ['msvc', 'clang-cl', 'intel-cl']
+  export_dll_args = ['-DBAR_DLL_EXPORTS']
+  import_dll_args = ['-DBAR_DLL_IMPORTS']
+else
+  export_dll_args = []
+  import_dll_args = []
+endif
```

Comment:
I think we decided to use 4 spaces indentation for all `meson.build` files.

Reply:
Sorry about all those - syncing from SciPy with its 2-space indent made those sneak in.